src/scrape_up/flexjobs/flexjobs.py

```diff
@@ -0,0 +1,132 @@
+import requests
+from bs4 import BeautifulSoup
+
+
+class FlexJobs:
+    """
+    A class to scrape job listings from FlexJobs website based on search query and location.
```

Comment:
The docstring of the class and the methods are not properly formatted. You can check modules like `GitHub` and `IMDB` for a reference to how these modules have their docstring formatted.

Reply:
Okay i will fix that out.

Reply:
@nikhil25803 changed the docstrings could you check now